src/api/db/data/20200527135426_request_delegate_attribute.rb

```diff
@@ -6,10 +6,10 @@ def self.up
 
     AttribTypeModifiableBy.reset_column_information
 
-    at = AttribType.create(attrib_namespace: ans, name: 'DelegateRequestTarget')
+    at = AttribType.find_or_create_by(attrib_namespace: ans, name: 'DelegateRequestTarget')
 
     role = Role.find_by_title('maintainer')
-    AttribTypeModifiableBy.create(role_id: role.id, attrib_type_id: at.id)
```

Comment:
This could have created duplicates in the database, but happily for us, it couldn't happen since `AttribType.create` always failed with `Duplicate entry` before.